app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/HttpDesBusinessPartnerRecordApiConnector.scala

```diff
@@ -1,68 +0,0 @@
-/*
- * Copyright 2017 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.hmrc.agentsubscriptionfrontend.connectors
-
-import java.net.URL
-import javax.inject.{Inject, Named}
-
-import com.google.inject.Singleton
-import play.api.libs.json.Json.parse
-import uk.gov.hmrc.agentsubscriptionfrontend.models._
-import uk.gov.hmrc.play.http._
-import uk.gov.hmrc.play.http.logging.Authorization
-
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-
-trait DesBusinessPartnerRecordApiConnector {
-
-
-  def getBusinessPartnerRecord(utr: String)(implicit hc: HeaderCarrier): Future[DesBusinessPartnerRecordApiResponse]
-
-}
-
-@Singleton
-class HttpDesBusinessPartnerRecordApiConnector @Inject()(@Named("des-baseUrl") desBaseUrl: URL,
-                                                         @Named("des.authorization-token") authorizationToken: String,
-                                                         @Named("des.environment") environment: String,
-                                                         httpGet: HttpGet) extends DesBusinessPartnerRecordApiConnector {
-
-  def getBusinessPartnerRecord(utr: String)(implicit hc: HeaderCarrier): Future[DesBusinessPartnerRecordApiResponse] = {
-    val url: String = bprUrlFor(utr)
```

Comment:
I'm not a big fan of explicit type annotations. Less code the better imho. But if the team's policy is to explicitly annotate then that is fine.